MicroProfile-Metrics/tck-runner/pom.xml

```diff
@@ -145,7 +164,7 @@
             <activation>
                 <property>
                     <name>payara.version</name>
-                    <value>/^5\.193(\.\d*)?(-SNAPSHOT)?/</value>
+                    <value>/5\.19[^1].*|5\.201.*/</value>
```

Comment:
Wouldn't this now match against 5.192? Conflict with profile above.

Reply:
Also 5.201 uses metrics 2.2 not 2.0.1

Reply:
You are right, I did not think about the default and the version interaction. This needs another profile for 5.201

Reply:
Turns out we supported metrics 2.2 from 5.194, so the new profile needs to match both 5.194 and 5.201, with 5.193 supporting metrics 2.0

Reply:
Thanks for looking that up